strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenIndexControllerTest.java

```diff
@@ -295,7 +295,7 @@ public void shouldDownloadProxyRepositoryIndex(@MavenRepository(repositoryId = R
                .peek()
                .then()
                .statusCode(HttpStatus.OK.value())
-               .contentType("application/x-gzip")
+               .contentType(org.carlspring.strongbox.net.MediaType.APPLICATION_X_GZIP_VALUE)
```

Comment:
import it directly ->` import static org.carlspring.strongbox.net.MediaType.APPLICATION_X_GZIP_VALUE;`

Reply:
@ze12augusto that's the last thing from my side :)

Reply:
I agree. It will make things look cleaner.

Reply:
Its done